demo-simple/src/main/java/com/novoda/downloadmanager/demo/DemoApplication.java

```diff
@@ -7,9 +7,11 @@
 import com.facebook.stetho.Stetho;
 import com.novoda.downloadmanager.DownloadManagerBuilder;
 import com.novoda.downloadmanager.LiteDownloadManagerCommands;
+import com.novoda.downloadmanager.StorageRequirementRuleFactory;
 
 public class DemoApplication extends Application {
 
+    private static final int TWO_HUNDRED_MB_IN_BYTES = 200000000;
```

Comment:
shouldn't this be 209,715,200?

Reply:
oh very true 😂 I was using integer version to make it easier in the tests but shouldn't be this here 😂